zend_operator.go

```diff
@@ -96,6 +96,7 @@ func ConvertToString(value any) (string, error) {
 		if result, ok := value.(toStringAble); ok {
 			return result.toString(), nil
 		}
+		return "Object", nil
```

Comment:
In PHP 5.6, when dealing with an object type, the `_convert_to_string` function calls the `convert_object_to_type `function, which in turn invokes the `zend_std_cast_object_tostring` function. This process eventually hits a switch statement with a [`is_string` case](https://github.com/php/php-src/blob/4b8f72da5dfb201af4e82dee960261d8657e414f/Zend/zend_object_handlers.c#L1563-L1596), where we encounter the following four scenarios:

* If an object's `__toString` function throws an exception, it returns FAILURE.
* If the `__toString` function's return value is a string, it returns SUCCESS.
* If the `__toString` function's return value is not a string, it returns FAILURE.
* All other cases result in FAILURE.

Returning to the logic within the `convert_object_to_type function`, if the [result from zend_std_cast_object_tostring is FAILURE, an ERROR level log is output, and ZEND_ERROR is triggered](https://github.com/php/php-src/blob/4b8f72da5dfb201af4e82dee960261d8657e414f/Zend/zend_operators.c#L337-L340). Consequently, in the `_convert_to_string function`, objects that cannot be converted to string do not return as "object", but rather the process terminates. Therefore, I think that in replicating PHP behavior accurately, the `ConvertToString` function in gophplib should appropriately handle these four cases.